source/Glimpse.Site/Views/Shared/_Layout.cshtml

```diff
@@ -13,6 +13,18 @@
 
     <title>@ViewBag.Title - the open source diagnostics platform of the web</title>
     @Styles.Render("~/content/sitecss")
+
+    <script type="text/javascript">
+    var appInsights=window.appInsights||function(config){
+        function s(config){t[config]=function(){var i=arguments;t.queue.push(function(){t[config].apply(t,i)})}}var t={config:config},r=document,f=window,e="script",o=r.createElement(e),i,u;for(o.src=config.url||"//az416426.vo.msecnd.net/scripts/a/ai.0.js",r.getElementsByTagName(e)[0].parentNode.appendChild(o),t.cookie=r.cookie,t.queue=[],i=["Event","Exception","Metric","PageView","Trace"];i.length;)s("track"+i.pop());return config.disableExceptionTracking||(i="onerror",s("_"+i),u=f[i],f[i]=function(config,r,f,e,o){var s=u&&u(config,r,f,e,o);return s!==!0&&t["_"+i](config,r,f,e,o),s}),t
+    }({
+        instrumentationKey:"a052ff85-1c8e-4175-97b1-0b4fb9b09600"
+    });
+
+    window.appInsights=appInsights;
+    appInsights.trackPageView();
+    </script>
+
```

Comment:
Couple of things. Does this need to go in the head? Also does it make sense for this script to be wrapped up in a bundle called `~/bundles/headsitejs` like what we have [here](https://github.com/Glimpse/Glimpse.Site/blob/9c03bd60c08497ace264d35fd3df0e74a8003ada/source/Glimpse.Site/App_Start/BundleConfig.cs#L11-L14)